packages/embark/src/cmd/cmd_controller.js

```diff
@@ -229,8 +233,12 @@ class EmbarkController {
       }
     ], function (err, _result) {
       if (err) {
-        engine.logger.error(err.message);
-        engine.logger.info(err.stack);
+        if (err.message && err.stack) {
+          engine.logger.error(err.message);
+          engine.logger.info(err.stack);
+        } else {
+          engine.logger.error(err);
+        }
```

Comment:
This check here is needed because **sometimes** we're getting an actual `Error` sometimes we're getting just a string (error message).

Hopefully in the future we have this streamlined and always emit a proper error type (see #911)